src/ui/views/Dashboard/components/EcologyPopup/index.tsx

```diff
@@ -16,7 +17,7 @@ interface Props {
 export const EcologyPopup = ({ visible, onClose }: Props) => {
   const { t } = useTranslation();
   const history = useHistory();
-  const chainList = [DBK_CHAIN_ID]
+  const chainList = [DBK_CHAIN_ID, SONIC_TESTNET_CHAIN_ID]
```

Comment:
You can add sonic testnet into https://github.com/RabbyHub/Rabby/blob/develop/src/ui/views/Ecology/constants.tsx and the entry will be works, please reset all changes in this file and just add your chain